website/.env.example

```diff
@@ -1,6 +1,7 @@
 CONFIG_DIR=../backend/src/main/resources/
 BACKEND_URL=http://localhost:8080
 DASHBOARDS_ENVIRONMENT=dashboards-dev
+LOG_DIR=./logs
```

Comment:
I think we should also add the log level here, since we have no other place to document it.